src/python/azimuthalaverage.py

```diff
@@ -565,8 +565,41 @@ def azimuthal_average(
 
 
 class AAWriter(Writer):
-    """FastDDM azimuthal average writer class.
-    Inherits from ``Writer``.
+    """Azimuthal average writer class. Inherits from ``Writer``.
+
+    It adds the unique method ``write_obj``.
+
+    Defines the functions to write :py:class:`AzimuthalAverage` object to binary file.
+
+    The structure of the binary file is the following:
+
+    Header:
+
+    * bytes 0-1: endianness, string, utf-8 encoding [``"LL"`` = 'little', ``"BB"`` = 'big']
+    * bytes 2-3: file identifier, 16-bit integer, unsigned short [``22`` for azimuthal average]
+    * bytes 4-5: file version, pair of 8-bit integers as (major_version, minor_version), unsigned char
+    * byte 6: dtype, string, utf-8 encoding [``"d"`` = float64, ``"f"`` = float32]
+    * bytes 7-14: data height, 64-bit integer, unsigned long long
+    * bytes 15-22: data width, 64-bit integer, unsigned long long
+    * bytes 23-30: extra slices, 64-bit integer, unsigned long long
+    * byte 31: error flag, 8-bit integer,unsigned char [``0`` if error is None, ``1`` otherwise]
```

Comment:
be more explicit that here `error` means the standard deviation and not a corruption error in the file